opensearch-operator/pkg/builders/dashboards.go

```diff
@@ -208,13 +208,21 @@ func NewDashboardsConfigMapForCR(cr *opsterv1.OpenSearchCluster, name string, co
 	}
 }
 
-func NewDashboardsSvcForCr(cr *opsterv1.OpenSearchCluster) *corev1.Service {
+func NewDashboardsSvcForCr(cr *opsterv1.OpenSearchCluster, customLabels map[string]string) *corev1.Service {
 	var port int32 = 5601
 
-	labels := map[string]string{
+	metadataLabels := map[string]string{
+		"opensearch.cluster.dashboards": cr.Name,
+	}
+	selectorLabels := map[string]string{
```

Comment:
Do we need this selectorLabels ?

Reply:
This is useful when you want to add a label to the service’s metadata that doesn’t exist on any pods, thereby keeping metadata labels separate from the selector labels used for pod targeting